package.json

```diff
@@ -2,14 +2,19 @@
   "name": "iopipe",
   "version": "0.3.1",
   "description": "IOpipe Lambda Analytics & Tracing Agent",
-  "main": "index.js",
+  "main": "dist/iopipe.js",
   "scripts": {
-    "test": "istanbul cover jasmine --captureExceptions"
+    "test": "istanbul cover jasmine --captureExceptions",
+    "build": "webpack --progress --profile --colors",
+    "prepublish": "webpack --progress --profile --colors"
```

Comment:
Upvote for prepublishOnly, since prepublish also has different behavior across versions
>IMPORTANT: As of npm@5, prepublish will only be run for npm publish. This will make its behavior identical to prepublishOnly, so npm@6 or later may drop support for the use of prepublishOnly, and then maybe we can all forget this embarrassing thing ever happened.